readme.md

```diff
@@ -103,6 +103,7 @@
 - [RubyMotion](https://github.com/motion-open-source/awesome-rubymotion) - Write cross-platform native apps for iOS, Android, macOS, tvOS, and watchOS in Ruby.
 - [Smart TV](https://github.com/vitalets/awesome-smart-tv) - Create apps for different TV platforms.
 - [GNOME](https://github.com/Kazhnuz/awesome-gnome) - Simple and distraction-free desktop environment for Linux.
+- [ROS2](https://github.com/fkromer/awesome-ros2) - Robot Operating System 2.0
```

Comment:
Can you make the title be `Robot Operating System 2.0` and use the description field to succinctly describe what it is?

Reply:
Done